meta/main.yml

```diff
@@ -14,7 +14,8 @@ galaxy_info:
   platforms:
     - name: Windows
       versions:
-        - 8.1
-        - 10
+        - 2012R2
+        - 2016
+        - 2019
```

Comment:
Do we have to specify `versions` at all? I'd rather be generic instead of pretending that this role works on these versions of Windows Server (since we haven't actually tested that).

Reply:
I'm not sure whether `versions` is optional, but we could do:

```yaml
platforms:
  - name: Windows
    versions:
      - all
```

We do this in several other roles of ours.

Reply:
That does look better to me.

Reply:
Fixed in ad27aa4.